src/geckomat/utilities/updateGECKOdoc.m

```diff
@@ -0,0 +1,33 @@
+function updateGECKOdoc()
```

Comment:
Thanks for adding this @edkerk, I was too lazy to add it myself. Also, I kind of wanted to avoided adding it, since it is essentially the same as in RAVEN with the exception of the base directory and the folders to be avoided. A more reusable solution would be to generalise the function in RAVEN to take these as parameters, so then one could call a function to generate documentation for any "parent" folder that is desired. In my view, this PR achieves its purpose with or without this function.
This PR can proceed as-is, but if you agree @edkerk perhaps you could open an issue in RAVEN.

Reply:
I agree that a generic function would be good, but it's very low priority tbh.